### DISCUSSION TITLE
Prioritize home row keys for `goto_word`

### DISCUSSION THREAD

**Discussion by hakan-demirli** [helix-editor/helix#11568] [General] [answered]
I have just started to use `goto_word` more frequently and one thing that really bothers me is the labeling order of the jump words.
For example, I want to jump to `lastModified` time and I need to press `a` and `p` which are located at pinky positions on all QWERTY keybards.

![image](https://github.com/user-attachments/assets/7f049903-5098-41b3-ac1c-3ce379252795)

It would be quite convenient if the words closer to the cursor are assigned home row keys near the center of the keyboard.

For example for nearest words: `dd`, `ff`, `gg`, `hh`, `jj`, `kk`, `tt`, `yy`, `fg`, `hj`, `ee`, `ii` ...

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/discussions/10017 has a suggested alphabet. Something like this would have to be addressed by a plugin in the future.

**Comment by hakan-demirli** [comment]
```toml
jump-label-alphabet = "dfghjkieruwonvls"
```
